src/test/bench/msgsend-ring-mutex-arcs.rs

```diff
@@ -107,9 +107,9 @@ fn main() {
 
     // all done, report stats.
     let num_msgs = num_tasks * msg_per_task;
-    let rate = (num_msgs as f64) / (dur.num_milliseconds() as f64);
+    let rate = (num_msgs as f64) / (dur.secs() as f64);
 
-    println!("Sent {} messages in {} ms", num_msgs, dur.num_milliseconds());
```

Comment:
Because there’s no num_mulliseconds anymore and Duration has a proper `Display` impl now.